cmd/gaia/lcd_test/helpers_test.go

```diff
@@ -353,7 +353,7 @@ func InitializeTestLCD(t *testing.T, nValidators int, initAddrs []sdk.AccAddress
 		node.Stop() //nolint:errcheck
 		node.Wait()
 		lcd.Close()
-		require.NoError(t, os.RemoveAll(config.RootDir))
+		os.RemoveAll(config.RootDir)
```

Comment:
Do we expect the error here to be non-deterministic? Why remove?

Reply:
Yes, that's correct. Some directories may not be removed because there are processes that are still writing files into them at the time of the removal. We shall replace the lcd tests in the near future, I wouldn't spend too much time into investigating this small thing.